plugins/queue/smtp_forward.js

```diff
@@ -6,10 +6,19 @@
 
 var smtp_client_mod = require('./smtp_client');
 
+// exported so tests can override config dir
+exports.get_net_utils = function () {
+    return require('haraka-net-utils');
+}
```

Comment:
What we often do in a case like this is instead:

```js
exports.net_utils = require('haraka-net-utils'); 
```

That lets you override net_utils config dir from tests and without having to run the node module loader every time that code is run in production (many many many millions of times).